README.md

```diff
@@ -1,38 +1,19 @@
 # SIG Release
 
-## Mission
-- Production of high quality Kubernetes releases on a reliable schedule
-- Ensure there is a [consistent group of community members][1.6-retro] in place to support the release process
-- Provide guidance and tooling to facilitate the production of automated releases
-- Serve as a tightly integrated partner with other SIGs to empower SIGs to integrate their repositories into the release process
+## Charter
 
-## Join us!
-- [Google group](https://groups.google.com/forum/#!forum/kubernetes-sig-release)
-- [Slack channel](https://kubernetes.slack.com/messages/C2C40FMNF/)
-- [Events and meetings calendar](https://calendar.google.com/calendar/embed?src=coreos.com_regcvcrgvq98lua2ikijg1g1uk%40group.calendar.google.com&ctz=America/Los_Angeles)
-- [Meeting agenda and notes](https://docs.google.com/document/d/1Fu6HxXQu8wl6TwloGUEOXVzZ1rwZ72IAhglnaAMCPqA/edit) ([read-only archive](https://docs.google.com/a/google.com/document/d/e/2PACX-1vRsGZTlmggTI5T4ysYGlaJiaW3vgnrc_dlv-B1nnt69gU2Xt6vzRpZDMm-7driIj14T1DraicGzkAHr/pub))
+The [charter] defines the scope and governance of the Release Special Interest Group.
 
-## Responsibilities
-- Ensuring high quality Kubernetes releases
-  - Define and staff release roles to manage the resolution of release blocking criteria
-  - Define and drive development processes (e.g. merge queues, cherrypicks) and release processes 
-    (e.g. burndown meetings, cutting beta releases) with the intent of meeting the release schedule
-  - Manage the creation of release specific artifacts including
-    - Code branches
-    - Binary distributions
-    - Release notes
-- Continually improving release and development processes
-  - Work closely with SIG Contributor Experience to define and build tools to facilitate release process (e.g. dashboards)
-  - Work with downstream communities responsible for packaging Kubernetes releases
-  - Work with other SIGs to agree upon the responsibilities of their SIG with respect to the release
-  - Define and collect metrics related to the release in order to measure progress over each release
-- Collaboration with downstream communities which build artifacts from Kubernetes releases
-  - Ensure the appropriate level of integration with publishing the OSS build artifacts
+## Join us!
```

Comment:
> Hey, this deletes all meeting and contact information, and that meeting information is NOT replicated in the charter. IMHO, we should leave meeting and contact information in the README.

@jberkus -- the contact info is here. It just got pushed a little lower in the diff.